tasks/requirements.yml

```diff
@@ -1,9 +0,0 @@
-- name: check if anonymous steam account has no password
```

Comment:
Why remove this in this PR? I'll be happy to discuss this in another PR or issue if you consider this useless.

Reply:
Because tasks/requirements.yml is never called, or did I miss something?

Reply:
readded in cfc768e312d2da16c1ca7103f8f1236d56c57a08